src/client/hooks/useLinkedData.js

```diff
@@ -1,25 +1,34 @@
-import { useEffect, useState } from 'react'
+import { useEffect, useState, useReducer } from 'react'
 
 import { format } from '../../modules/3box-LD'
+import {
+  boxReducer,
+  initialState,
+  fetchedPublicProfileError,
+} from '../stateManagers/box'
 
-const useLinkedData = boxes => {
+const useLinkedData = () => {
   // const { connectedAccount } = useAragonApi()
   const connectedAccount = ''
```

Comment:
Should we use the `connectedAccount`? Can't remember why this was commented out. Do you know?

Reply:
As [mentioned in another PR](https://github.com/AutarkLabs/aragon-profile/pull/62#discussion_r302036209), `aragon-profile` will be included in the Aragon Wrapper itself. `useAragonApi` is for apps that are embedded within the wrapper via an iframe. Is there an alternative way to fetch fetch this data?

Reply:
connectedAccount is passed via props (`account`) from Wrapper itself. I improved this bit, so variable is passed correctly... also, cleaned up account vs connectedAccount vs ethereumAddress naming - the same data, different names, depending on context.